lib/registries/addon/branded/new/template.hbs

```diff
@@ -16,50 +16,83 @@
         </layout.heading>
         <layout.main local-class='main'>
+            <p local-class='InformationText'>
+                {{t 'registries.new.provider_info' provider=this.model.name htmlSafe=true}}
+            </p>
             <form data-test-new-registration-form local-class='registrationForm'>
-                <label
-                    data-test-project-select
-                    local-class='selectLabel'
-                >
+                <label local-class='selectLabel StepOneContainer'>
                     <p local-class='stepLabel'>
                         {{t 'registries.new.step_one'}}
                     </p>
-                    <h2
-                        data-test-project-select-title
-                        local-class='projectHeading'
-                    >
-                        {{t 'registries.new.select_project'}}
-                        <span local-class='required' aria-label={{t 'registries.new.required'}}>*</span>
-                    </h2>
-                    <PowerSelect
-                        @ariaLabel={{this.selectedProject.title}}
-                        @placeholder={{t 'registries.new.select_project_placeholder'}}
-                        @options={{this.projectOptions}}
-                        @search={{perform this.projectSearch}}
-                        @selected={{this.selectedProject}}
-                        @onchange={{this.updateSelectedProject}}
-                        as |option|
-                    >
-                        {{option.title}}
-                    </PowerSelect>
+                    <fieldset local-class='RadioFieldset'>
+                        <legend local-class='RadioLegend'>
+                            {{t 'registries.new.step_one_project_heading'}}
+                        </legend>
+                        <input
+                            data-test-has-project-button
+                            local-class='RadioElement RadioInput'
+                            type='radio'
+                            name='HasProject'
+                            value='yes'
+                            onclick={{fn (mut this.isBasedOnProject) true}}
+                            checked={{this.isBasedOnProject}}
+                        >
+                        <label
+                            local-class='RadioElement RadioLabel {{if this.isBasedOnProject 'IsChecked'}}'
+                            for='HasProject'
+                        >
+                            {{t 'registries.new.yes'}}
+                        </label>
+                        <input
+                            data-test-no-project-button
+                            local-class='RadioElement RadioInput'
+                            type='radio'
+                            name='no-project'
+                            value='no'
+                            onclick={{fn (mut this.isBasedOnProject) false}}
+                            checked={{this.isBasedOnProject}}
```

Comment:
Should this be `{{not this.isBasedOnProject}}`?

Reply:
Oh, you're right! Fixed it.